### PR TITLE
feat(sweeps): Move run configs from entrypoint into params

### PR DESCRIPTION
**Pull request by hu-po**
Fixes WB-10661

Description
-----------
Moves run configs from going through entrypoint in `launch_add` to going through the `params` kwarg.

Testing
-------
How was this PR tested?

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Review by KyleGoyette — CHANGES_REQUESTED** [review]
If you have some time could you elaborate on what couldn't be done by making this change? Im concerned baout loss of functionality

File: wandb/sdk/launch/sweeps/scheduler_sweep.py [hunk 1]
```diff
@@ -144,22 +141,10 @@ def _run(self) -> None:
...
-        wandb.termlog(f"{LOG_PREFIX}Saving params to {sweep_param_path}")
-        wandb_lib.config_util.save_config_file_from_dict(sweep_param_path, run.args)
-        # Construct entry point using legacy sweeps utilities
-        command_args = LegacySweepAgent._create_command_args({"args": run.args})["args"]
-        entry_point = ["python", run.program] + command_args
         _ = self._add_to_launch_queue(
-            entry_point=entry_point,
             run_id=run.id,
+            entry_point=["python", run.program] if run.program else None,
+            params=run.args,
```
[the remark this answers is not in the document]
Comment (hu-po) [reply wandb/sdk/launch/sweeps/scheduler_sweep.py:147]: So through [here](https://github.com/wandb/wandb/blob/44270677725ee24fc0d8ef4151b4c9c99f1e69c5/wandb/sdk/launch/utils.py#L122)?

[the remark this answers is not in the document]
Comment (hu-po) [reply wandb/sdk/launch/sweeps/scheduler_sweep.py:147]:
```
...
      queued_run = launch_add(
          run_id=run_id,
          entry_point=entry_point,
          # params=params,
          launch_config={"overrides": {"run_config": run.args},
          ...
```

**Review by KyleGoyette — APPROVED** [review]
🚢 For future readers: we talked about the implications of moving this out of the args and into run config. Users constant args will be supported by jobs which gets rid of the loss of any functionality.

**Pull request merged**